assets/helpers/utilities.js

```diff
@@ -19,14 +19,13 @@ function roundDp(num: number, dps: number = 2) {
 }
 
 // Generates the "class modifier-class" string for HTML elements
-function generateClassName(className: string, modifierClass: ?string): string {
-  let response = className;
-
-  if (modifierClass) {
-    response = `${className} ${className}--${modifierClass}`;
-  }
+function classNameWithModifiers(className: string, modifiers?: string[] = []): string {
+  return modifiers.reduce((acc, modifier) => `${acc} ${className}--${modifier}`, className);
+}
 
-  return response;
+// Generates a className with an optional modifier (common pattern convenience function).
+function classNameWithOptModifier(className: string, modifier: ?string): string {
```

Comment:
Just a question about this function, Would it make sense for a developer to call it with modifier equal to `null`? As far as I understand, if we call it with a `null` modifier it will returns the same class function.  If that is the case, probably is better to make the modifier non-optional.